pkg/tdex-market-loader/client.go

```diff
@@ -122,7 +161,7 @@ func (t *tdexMarketLoaderService) FetchPrice(
 		Asset:  market.BaseAsset,
 	})
 	if err != nil {
-		return nil, err
+		return decimal.NewFromInt(0), decimal.NewFromInt(0), err
```

Comment:
we can use  `decimal.Zero` here:
```suggestion
		return decimal.Zero, decimal.Zero, err
```

Reply:
Done